src/js/card.js

```diff
@@ -0,0 +1,151 @@
+import { playAudio } from './function'
+
+export class Card {
+  constructor(id, cardData) {
+    this.id = id
+    this.cardData = cardData
+  }
+
+  buildCardContainer(selector, className) {
+    const block = document.querySelector(selector)
+    const card = document.createElement('div')
+    const cover = document.createElement('div')
+    const infoContainer = document.createElement('div')
+    const cardTitle = document.createElement('h2')
+    const indicator = document.createElement('div')
+
+    card.classList.add(className)
+    cover.classList.add('card__cover')
+    infoContainer.classList.add('card__info_block')
+    cardTitle.classList.add('info_block__title')
+    indicator.classList.add('info_block__indicator')
+
+    card.setAttribute('id', this.id)
+
+    block.appendChild(card)
+
+    card.appendChild(cover)
+
+    card.appendChild(infoContainer)
+    infoContainer.appendChild(cardTitle)
+    infoContainer.appendChild(indicator)
+  }
+
+  renderCard() {
+    new RenderCard(this.cardData, this.id).render()
+  }
+
+  renderSubCard() {
+    const subCard = new RenderSubCard(this.cardData, this.id)
+    subCard.render()
+    subCard.rotate()
+    subCard.speech()
+  }
+}
+
+class RenderCard {
+  constructor({ category, cover, words }, id) {
+    this.cardData = { category, cover, words }
+    this.id = id
+  }
+
+  render() {
+    const card = document.getElementById(this.id)
+    const quantityWords = card.querySelector('.info_block__quantity')
+
+    card.setAttribute('name', this.cardData.category)
+    card.classList.add('card')
+
+    const infoContainer = card.querySelector('.card__info_block')
+    const cover = card.querySelector('.card__cover')
+    const rotate = card.querySelector('.rotate')
+
+    if (card.querySelector('.info_block__subtitle')) {
```

Comment:
I'd like to add state with setting of card and use it. Use querySelector is not a good way

Reply:
+++